packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -23,6 +21,7 @@ class Chat extends Widget<Properties> {
       ...super._getDefaultOptions(),
       title: '',
       items: [],
+      user: undefined,
```

Comment:
There are no tests.

Reply:
Was dealing with CI at the time, hadn't gotten around to finishing it yet. It's done now

[af552e68f7ca2ad08a6f510a966f83272b925c78]